truck_inspection_client/src/truck_inspection_client.cpp

```diff
@@ -84,7 +84,8 @@ namespace truck_inspection_client
             mo_msg.strategy = this->INSPECTION_STRATEGY;
             std::string ads_auto_status = this->ads_engaged_ ? "Engaged" : "Not Engaged";
             std::string ads_health_status = ads_system_alert_type_ ;
-            std::string params = boost::str(boost::format("vin_number:%s,license_plate:%s,carrier_name:%s,carrier_id:%s,weight:%d,ads_software_version:%s,date_of_last_state_inspection:%s,date_of_last_ads_calibration:%s,pre_trip_ads_health_check:%s,ads_health_status:%s,ads_auto_status:%s,iss_score:%d,permit_required:%s")
+            //std::string params = boost::str(boost::format("vin_number:%s,license_plate:%s,carrier_name:%s,carrier_id:%s,weight:%d,ads_software_version:%s,date_of_last_state_inspection:%s,date_of_last_ads_calibration:%s,pre_trip_ads_health_check:%s,ads_health_status:%s,ads_auto_status:%s,iss_score:%d,permit_required:%s")
```

Comment:
Please add a comment above that you've commented this out to put a temporary fix due to github issue#...

Reply:
Added https://github.com/usdot-fhwa-stol/carma-platform/pull/882/commits/b01c1401bfb68bf4b2997afbed5adbff4b7d593e